### ISSUE TITLE
Accessibility testing report, required fixes

### ISSUE THREAD

**Issue by ahuseyn** [wpengine/faustjs.org#272]
## Methodology

For the automated accessibility testing website was crawled and available URLs gathered using **crawlee**, the npm tool. A total of 57 URLs were gathered. For the automated testing **pa11y** npm tool has been used. Failures then grouped under WCAG identifiers, which formed two major groups:
- Guideline1_4.1_4_3.G18.Fail - Insufficient contrast at this conformance level.
- Guideline2_4.2_4_2.H25.1.NoTitleEl - A title should be provided for the document, using a non-empty title element in the head section
Most of the failures represent global level issues. Issues concerning individual pages are discussed under the results section.

For the manual testing 12 sample URLs were gathered. Samples chosen to represent the different content types (image, list, link, etc.). Manual test conducted using **Accessibility Insights for Web** chrome extension. In the extension menu **Quick assess** option used and for each page 10 accessibility criteria assessed.

For the text-to-speech test macOS VoiceOver used. Basic navigation testing conducted for each of the 12 sample urls. Testing started at the beginning of the page and ended at the end while navigation through the page using **VoiceOver** assistant.

## Tools used
- Node.js
- Pa11y
- Crawlee
- Accessibility Insights for Web chrome extension
- macOS VoiceOver

### Global issues
- [x] Missing skip link to header
- [x] Missing skip link to navbar
- [x] ⌘K button in the searchbar has insufficient color contrast
- [x] "Powered by Faust.js & WP Engine's Headless platform" text in the footer has insufficient color contrast
- [x] "Footer © 2013-2025 WP Engine, Inc. All rights reserved" text in the footer has insufficient color contrast
- [x] Breadcrumbs have insufficient color contrast
- ~~Comments inside `<pre><code></code></pre>` have insufficient color contrast~~
- [x] Docs section, left sidebar: `<ul>` cannot be a direct child of `<ul>`, should be wrapped with `<li>`
-  ~~Docs section, right sidebar: `<h2>` font style is different than other `<h2>` styles~~
- [x] Usability: docs section, left sidebar jumps to top when you click an item
- [x] Docs left sidebar each list iteam wrapped with ul tag. It creates an issue for the VoiceOver

#### 404 page
- [x] Has no title

#### [Privacy Policy](https://toolkit.faustjs.org/privacy-policy) 
- [x] Has no title

Links below have no accessible names:
- [ ] <a href="https://wpengine.com/legal/privacy/" target="_blank" rel="noreferrer noopener">https://wpengine.com/legal/privacy/</a>
- [ ] <a href="https://wpengine.com/legal/dpa/" target="_blank" rel="noreferrer noopener">https://wpengine.com/legal/dpa/</a>
- [ ] <a href="http://www.networkadvertising.org/choices/" target="_blank" rel="noreferrer noopener">click here</a>
- [ ] <a href="http://www.youronlinechoices.eu/" target="_blank" rel="noreferrer noopener">here</a>
- [ ] <a href="https://www.privacyshield.gov/list" target="_blank" rel="noreferrer noopener">https://www.privacyshield.gov/list</a>
- [ ] <a href="https://feedback-form.truste.com/watchdog/request" target="_blank" rel="noreferrer noopener">https://feedback-form.truste.com/watchdog/request</a>
- [ ] <a href="https://www.privacyshield.gov/article?id=How-to-Submit-a-Complaint" target="_blank" rel="noreferrer noopener">https://www.privacyshield.gov/article?id=How-to-Submit-a-Complaint</a>

#### [Blog](https://toolkit.faustjs.org/blog/)
- [x] Load more button has insufficient color contrast (between bg and text, suggested bg: #7e63b6)
- #283

#### [Deploy your app](https://toolkit.faustjs.org/docs/explanation/deploy-your-app/)
- [x] Text with link: "Additionally, you can explore the Headless Platform framework guide for Next.js apps `here`." (link on `here` should be added to `framework guide`)
- [x] For further reference, please check out the Headless Platform framework guide docs on Deploying Next.js `here`. (link on `here` should be added to `framework guide`)

## References:
- https://www.a11yproject.com/checklist/
- https://medium.com/civicactions/designing-solo-navigating-a11y-projects-checklist-with-automation-4fcc0ed18dbc
- https://accessibility-manual.dwp.gov.uk/best-practice/
